### PR TITLE
Create Role

### PR DESCRIPTION
**Pull request by tjoelsson**
## Summary of changes
- Create folders
- Generate .env config file
- Generate Nginx config file
## How to Test
1. `$ cd tests`
2. `$ vagrant up`
3. Don't see errors while provisioning (but if you do, try provisioning again)

/fyi: @wanleung @KixPanganiban: Please review.

### CODE REVIEW

**Comment by KixPanganiban** [comment]
Not sure what I'm doing wrong, but I'm getting this:

``` bash
$ vagrant provision
- downloading role 'common', owned by juwai
- downloading role from https://github.com/juwai/ansible-role-common/archive/v0.1.1.tar.gz
- extracting juwai.common to roles/juwai.common
- juwai.common was installed successfully
- downloading role 'nginx', owned by juwai
- downloading role from https://github.com/juwai/ansible-role-nginx/archive/master.tar.gz
- extracting juwai.nginx to roles/juwai.nginx
- juwai.nginx was installed successfully
[DEPRECATION WARNING]: The comma separated role spec format, use the yaml/explicit format instead..
This feature will be removed in a future release. Deprecation warnings can be
disabled by setting deprecation_warnings=False in ansible.cfg.
- dependency juwai.common already pending installation.
==> default: Running provisioner: ansible...
    default: Running ansible-playbook...
PYTHONUNBUFFERED=1 ANSIBLE_FORCE_COLOR=true ANSIBLE_HOST_KEY_CHECKING=false ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ControlMaster=auto -o ControlPersist=60s' ansible-playbook --connection=ssh --timeout=30 --limit="default" --inventory-file=/Users/kix/Projects/Juwai/ansible-role-laravel/tests/.vagrant/provisioners/ansible/inventory -v test.yml
No config file found; using defaults
ERROR! the role 'laravel' was not found in /Users/kix/Projects/Juwai/ansible-role-laravel/tests/roles:/Users/kix/Projects/Juwai/ansible-role-laravel/tests:/etc/ansible/roles

The error appears to have been in '/Users/kix/Projects/Juwai/ansible-role-laravel/tests/test.yml': line 16, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  roles:
    - laravel
      ^ here

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

Tried doing it again, same error.

**Comment by tjoelsson** [comment]
Sorry, forgot to commit the symlink

File: tests/inventory [hunk 1]
```diff
@@ -0,0 +1 @@
+localhost
```
Comment (KixPanganiban) [inline tests/inventory:1]: Specify a host here and in the Vagrantfile? As is, I'm getting:

``` bash
$ vagrant provision
[... removed for brevity...]

PLAY [default] *****************************************************************

TASK [setup] *******************************************************************
fatal: [default]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh.", "unreachable": true}
 [WARNING]: Could not create retry file 'test.retry'.         [Errno 2] No such
file or directory: ''


PLAY RECAP *********************************************************************
default                    : ok=0    changed=0    unreachable=1    failed=0

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

Comment (tjoelsson) [reply tests/inventory:1]: What is your Vagrant and Ansible versions?

Comment (KixPanganiban) [reply tests/inventory:1]:
```
$ vagrant --version
Vagrant 1.8.5
$ ansible --version
ansible 2.1.1.0
```

**Comment by tjoelsson** [comment]
@KixPanganiban After upgrading Ansible and Vagrant I also experienced that problem. It should be fixed now. Please try again.

**Comment by KixPanganiban** [comment]
This works now. 👍

**Comment by wanleung** [comment]
👍

**Pull request merged**